src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -115,16 +115,15 @@ internal static ProcessInfo CreateProcessInfo(int pid, ReusableTextReader reusab
                 reusableReader = new ReusableTextReader();
             }
```

Comment:
Nit: as long as you're here, this can be simplified to:
```C#
reusableReader ??= new ReusableTextReader();
```

Reply:
Done.